### PR TITLE
added @angular/router @types/geojson

### PR DESCRIPTION
**Pull request by GaryDaviesGaiaResources**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,10 +21,12 @@
     "@angular/http": "5.2.10",
     "@angular/platform-browser": "5.2.10",
     "@angular/platform-browser-dynamic": "5.2.10",
+    "@angular/router": "^6.0.3",
```
Comment (tony-gaia) [inline package.json]: Is this package necessary for a phone app?

Comment (GaryDaviesGaiaResources) [reply package.json]: Its necessary for the starting project

Comment (GaryDaviesGaiaResources) [reply package.json]: auth gary

Comment (GaryDaviesGaiaResources) [reply package.json]: auth guard

**Pull request closed without merging**